### ISSUE TITLE
help_color not working for multi_select

### ISSUE THREAD

**Issue by pjvleeuwen** [piotrmurach/tty-prompt#122]
### Describe the problem
The default help_color given to the Prompt constructor is used for `select`, but not for `multi_select`. On a first glance the intention was for `multi_select` to use this color, as 

### Steps to reproduce the problem
```
irb(main):001:0> require 'tty-prompt'
=> true
irb(main):002:0> @prompt = TTY::Prompt.new(help_color: :cyan)
=> #<TTY::Prompt: {:input=>#<IO:<STDIN>>, :output=>#<IO:<STDOUT>>, :prefix=>"", :active_color=>:green, :error_color=>:red, :enabled_color=>nil, :help_color=>:cyan}
irb(main):003:0> @prompt.select('Which option?', %i[first second third])
Which option? first
=> :first
irb(main):004:0> @prompt.multi_select('Which options?', %i[first second third])
Which options? first
=> [:first]
```

### Actual behaviour
`select` uses help_color configured, while `multi_select` always shows dark gray.

### Expected behaviour
The expected behaviour is for both classes to use `help_color`. I expect this, because `MultiList#render_header` seems to `decorate` based on inherited  `@help_color`.

### Describe your environment
* OS version: `Ubuntu 18.04.3 LTS`
* Ruby version: `ruby 2.6.5p114 (2019-10-01 revision 67812) [x86_64-linux]` (in docker, `FROM ruby:2.6.5`)
* TTY version:
```
gem list tty

*** LOCAL GEMS ***

tty-color (0.5.0)
tty-cursor (0.7.0)
tty-prompt (0.20.0)
tty-reader (0.7.0)
tty-screen (0.7.0)```

**Comment by piotrmurach** [comment]
Thanks for reporting! ❤️ Do you have time to submit PR to fix this?

**Comment by pjvleeuwen** [comment]
Not sure yet...

**Comment by pjvleeuwen** [comment]
I have not before worked on such ruby project. I just write scripts which I run with the ruby command from a docker container. Could you please write out some steps I need to take to get up and running? E.g. apt install this and that, then run this and that to build / run tests.

**Comment by slowbro** [comment]
This was fixed in #119 (9f72d62).
